### DISCUSSION TITLE
A method to solve the problem of Win10 style start menu not working in the latest version and internal components（解决win10风格start menu无法在最新版本和内部构件起作用的一个方法

### DISCUSSION THREAD

**Discussion by Gostronaut** [valinet/ExplorerPatcher#1957] [Show and tell]
#1909 
Some of my devices (which should be all) cannot use EP's start menu because they are versions that EP does not support. However, several devices still turned on normally, and during the exploration above, I found that I have installed start all backs on all devices that can use EP's start menu. When I installed start all back on all unsupported devices and switched to Win10 style, the EP start menu worked as expected，There is no conflict between them either

_我的部分设备（应该是所有）无法使用EP的start menu，因为他们是EP不支持的版本。但是仍然有几台设备正常打开了，在上面的探索中我发现了我在可以使用EP的start menu的设备上都安装了start all back。当我在所有不支持的设备上安装了start all back并切换到win10风格的时候，EP的start menu和预期一样工作，他们之间也没有任何冲突_

If your EP also has the same issue as mine, try installing the start all back and switching to the Win10 style. It will enable the start menu on the latest version

_如果你的EP也出现了和我一样的问题，试着安装一下start all back并切换到win10风格，他将在最新版上面启用start menu_

**Comment by Gostronaut** [comment]
I don't know the reason, but it really solved my problem.All of my Win11 devices can now open the EP start menu, with version numbers from 1772 to 2215

**Comment by pyrates999** [comment]
EP requires internet access for 2 reasons:

- downloading symbols
- checking for updates on github.com
